runbotics/runbotics-orchestrator-ui/src/src-app/components/InfoPanel/IterationSlide/IterationSlide.tsx

```diff
@@ -84,10 +84,9 @@ const IterationSlide: FC<IterationSlideProps> = ({
                                     {iterationGutter.iteratorElement && (
                                         <ReactJson
                                             src={{
-                                                iterator: JSON.parse(
+                                                iterator: 
```

Comment:
cant there be just null or json object?

Reply:
I see that you check existance of this element above, no need to add bullish coalescing